src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -79,7 +79,14 @@ private int Run(string[] args)
                     }
                     else if (ArgumentMatch(arg, "type"))
                     {
-                        types.Add(value);
+                        if(value != string.Empty)
+                        {
+                            string[] typelist = value.Split(';');
```

Comment:
>';' [](start = 60, length = 3)

Is making sgen handle the semi-colon the way we wish to handle this as this is different from sg? If it is, that's fine but if we want to use a comma in the tool, then we can do the change in msbuild.

Reply:
Yes, it is different from sg desktop version, which is using /type: type1 /type:type2